### PR TITLE
Fix plusminus label in genetic profile selector, and fix typo

### PR DESCRIPTION
**Pull request by adamabeshouse**
# Checks
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

### CODE REVIEW

File: src/shared/components/query/GeneSetSelector.tsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ export default class GeneSetSelector extends QueryStoreComponent<GeneSetSelector
 					onHide={() => this.store.showMutSigPopup = false}
 				>
 					<Modal.Header closeButton>
-						<Modal.Title>Recently Mutated Genes</Modal.Title>
+						<Modal.Title>Recurrently Mutated Genes</Modal.Title>
```
Comment (inodb) [inline src/shared/components/query/GeneSetSelector.tsx]: nice

File: src/shared/components/query/GeneticProfileSelector.tsx [hunk 2]
```diff
@@ -150,7 +150,7 @@ export default class GeneticProfileSelector extends QueryStoreComponent<{}, {}>
 		{
 			output.push(
 				<div key={output.length} className={styles.zScore}>
-					Enter a z-score threshold ±:
+					Enter a z-score threshold &plusmn;:
```
Comment (inodb) [inline src/shared/components/query/GeneticProfileSelector.tsx]: @adamabeshouse This doesn't seem to fix it. When I use the build from jitpack it still shows up as  Â±:. we should maybe do another CSS hack, @alisman ?

Comment (inodb) [inline src/shared/components/query/GeneticProfileSelector.tsx]: You can test like this in main project:
```
--- a/portal/pom.xml
+++ b/portal/pom.xml
@@ -93,9 +93,9 @@
             <configuration>
               <artifactItems>
                 <artifactItem>
-                  <groupId>com.github.cbioportal</groupId>
+                  <groupId>com.github.adamabeshouse</groupId>
                   <artifactId>cbioportal-frontend</artifactId>
-                  <version>f11ac015646604eb1d911f59eea9560d52fcc54b</version>
+                  <version>5f3778fa284b2fa446a943d7530cfef45f4e0d24</version>
                   <type>jar</type>
                   <outputDirectory>.</outputDirectory>
                   <excludes>*index*</excludes>
```

Comment (alisman) [inline src/shared/components/query/GeneticProfileSelector.tsx]: There are two corruption issues here.  THe close button in dialog and the plusmn.  The close button must be fixed with css hack b/c sp char is in bootstrap files.  I will do that as I've fixed it elsewhere.   The plusmn, let me just verify that entity doesn't fix it.  I would shocked if jitpack could really corrupt an HTML entity

**Pull request merged**